### ISSUE TITLE
Can't install Codemodify - Artifact not found

### ISSUE THREAD

**Issue by fap-** [vogellacompany/codemodify#9]
Eclipse is looking for a version of the plugin that isn't on bintray: https://dl.bintray.com/vogellacompany/junit-4-tools/features/

Tested this with several machines and eclipse versions. (oxygen and neon)

```
An error occurred while collecting items to be installed
session context was:(profile=SDKProfile, phase=org.eclipse.equinox.internal.p2.engine.phases.Collect, operand=, action=).
Artifact not found: http://dl.bintray.com/vogellacompany/junit-4-tools/plugins/de.simonscholz.codemodify_0.1.0.201612201144.jar.
http://dl.bintray.com/vogellacompany/junit-4-tools/plugins/de.simonscholz.codemodify_0.1.0.201612201144.jar
Artifact not found: http://dl.bintray.com/vogellacompany/junit-4-tools/features/de.simonscholz.codemodify.feature_0.1.0.201612201144.jar.
http://dl.bintray.com/vogellacompany/junit-4-tools/features/de.simonscholz.codemodify.feature_0.1.0.201612201144.jar
```

**Comment by SimonScholz** [comment]
Hi I've updated the code modify project and it is now available in version 1.2.0 right here: https://dl.bintray.com/vogellacompany/junit-4-tools/
Works for Neon and Oxygen.

Let me know, if you still have problems installing it.

**Comment by fap-** [comment]
Works now, ty.
